### ISSUE TITLE
[Bug]:  Cannot find module '@api.video/nodejs-client/lib/model/Video' 

### ISSUE THREAD

**Issue by Andrehatlo** [apivideo/api.video-nodejs-client#113]
### Version

"2.2.5",

### Environment that reproduces the issue

VSCode, Macbook Silicon, Chrome newest

### Is it reproducible in the example application?

Not tested

### Reproduction steps

Well this is something that happens only for this import. 
I installed the package as described: `npm install @api.video/nodejs-client --save`

Even after this i did a 
1. `rm -rf node_modules`
2. `rm -f package-lock.json`
4. `npm cache clean --force`

Then i ran 
1. `npm install`
2. `npm run build`

Then i get this error:
```
➜  kroony git:(main) ✗ npm run build

> kroony@1.0.0 build
> next build

info  - Linting and checking validity of types .warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config

./pages/index.tsx
17:11  Warning: 'state' is assigned a value but never used.  @typescript-eslint/no-unused-vars
52:11  Warning: 'response' is assigned a value but never used.  @typescript-eslint/no-unused-vars

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
info  - Linting and checking validity of types ...Failed to compile.

./components/Content/index.tsx:1:19
Type error: Cannot find module '@api.video/nodejs-client/lib/model/Video' or its corresponding type declarations.

> 1 | import Video from '@api.video/nodejs-client/lib/model/Video';
    |                   ^
  2 | import { useAuthContext } from '@components/Providers/Auth';
  3 | import React from 'react';
  4 | import ItemPreview from './ItemPreview';
```

### Expected result

That the package is available and runs.

### Actual result

Error: 
```
➜  kroony git:(main) ✗ npm run build

> kroony@1.0.0 build
> next build

info  - Linting and checking validity of types .warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config

./pages/index.tsx
17:11  Warning: 'state' is assigned a value but never used.  @typescript-eslint/no-unused-vars
52:11  Warning: 'response' is assigned a value but never used.  @typescript-eslint/no-unused-vars

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
info  - Linting and checking validity of types ...Failed to compile.

./components/Content/index.tsx:1:19
Type error: Cannot find module '@api.video/nodejs-client/lib/model/Video' or its corresponding type declarations.

> 1 | import Video from '@api.video/nodejs-client/lib/model/Video';
    |                   ^
  2 | import { useAuthContext } from '@components/Providers/Auth';
  3 | import React from 'react';
  4 | import ItemPreview from './ItemPreview';
```

**Comment by Andrehatlo** [comment]
This is how its included in my Next.js typescript application:

```
import React, { useEffect, useState } from 'react';
import { useRouter } from 'next/router';
import Navbar from '@components/Navbar';
import Video from '@api.video/nodejs-client/lib/model/Video';
```

**Comment by olivierapivideo** [comment]
Hi @Andrehatlo 

Thanks you for your feedback. We've just released a new version (v2.3.2) with a fix.

Feel free to reopen if it's not fine on your side. 

Sorry for the inconvenience!
